src/ServiceControl.Transports.Tests/ApprovalFiles/APIApprovals.ServiceControlTransport.approved.txt

```diff
@@ -46,6 +46,29 @@ namespace ServiceControl.Transports
         public System.Threading.Tasks.Task<ServiceControl.Transports.IQueueIngestor> InitializeQueueIngestor(string queueName, ServiceControl.Transports.TransportSettings transportSettings, System.Func<NServiceBus.Transport.MessageContext, System.Threading.Tasks.Task> onMessage, System.Func<NServiceBus.Transport.ErrorContext, System.Threading.Tasks.Task<NServiceBus.Transport.ErrorHandleResult>> onError, System.Func<string, System.Exception, System.Threading.Tasks.Task> onCriticalError) { }
         public virtual System.Threading.Tasks.Task ProvisionQueues(string username, ServiceControl.Transports.TransportSettings transportSettings, System.Collections.Generic.IEnumerable<string> additionalQueues) { }
     }
+    public class TransportManifest
+    {
+        public TransportManifest() { }
+        public System.Collections.Generic.List<ServiceControl.Transports.TransportManifestDefinition> Definitions { get; set; }
```

Comment:
As this is only ever deserialized and isn't intended to be added to at runtime, should this be an array?

Reply:
Done